django/river/settings/base.py

```diff
@@ -114,6 +117,7 @@
 # https://docs.djangoproject.com/en/3.1/topics/auth/customizing/#specifying-authentication-backends
 
 AUTHENTICATION_BACKENDS = [
+    "mozilla_django_oidc.auth.OIDCAuthenticationBackend",
```

Comment:
this does not break authentication for the current applications which does not implement this flow right ?

Reply:
This adds a backend to the list. Django will try each backend in definition order. Unless 1 backends raises (for instance the user has been blacklisted in the backend), every backend should be used. So users can still authenticated as before.
reminder: currently no auth is not plugged in the front and we do not restrict access to authenticated users.